src/parser/Parser.ts

```diff
@@ -314,6 +325,11 @@ export class Parser {
                 return this.namespaceStatement();
             }
 
+            if (this.check(TokenKind.At) && this.checkNext(TokenKind.Identifier)) {
```

Comment:
BrightScript uses the `identifier@otherIdentifier` to dereference xml attributes. Should we `checkPrevious()` to ensure there isn't a leading identifier before the `TokenKind.At`? It might not matter, but perhaps to verify, we should add a unit test to ensure a top-level `Identifier@otherIdentifier` doesn't trip this up.

Reply:
Should that happen? I'm matching when something "starts" with an @

Reply:
You're right, it probably won't happen. It was just a thought.